### ISSUE TITLE
Keyboard access to command history

### ISSUE THREAD

**Issue by mfikes** [mfikes/replete#18]
(cmd+Up)

**Comment by mfikes** [comment]
Currently the history is kept in an array-of-array structure and probably should be flattened. Then it would be easier to use it for this ticket.

**Comment by madvas** [comment]
so cmd+Up isn't implemented yet? It's super cool to use this app with wireless keyboard, this would be very nice feature

**Comment by mfikes** [comment]
Right. (I have no wireless keyboard so I'm frequently unaware of keyboard related deficiencies in Replete.)
